src/types/line.js

```diff
@@ -67,10 +67,10 @@ export default class LineAnnotation extends Element {
   // TODO: make private in v2
   labelIsVisible(useFinalPosition, chartArea) {
     const labelOpts = this.options.label;
-    if (!chartArea || !labelOpts || !labelOpts.enabled) {
+    if (!labelOpts || !labelOpts.enabled) {
       return false;
     }
-    return isLineInArea(this.getProps(['x', 'y', 'x2', 'y2'], useFinalPosition), chartArea);
+    return !chartArea || isLineInArea(this.getProps(['x', 'y', 'x2', 'y2'], useFinalPosition), chartArea);
```

Comment:
Oh right, the chartArea is optional parameter that makes this function call more precise. Could add some jsdoc to indicate the parameter is optional (and not just undefined in some scenario)

Reply:
I'll submit another PR, adding JSdoc